erigon-lib/chain/chain_config.go

```diff
@@ -703,6 +716,7 @@ type Rules struct {
 	IsPrague, isOsaka                                                                                                                                    bool
 	IsAura                                                                                                                                               bool
 	IsNormalcy                                                                                                                                           bool
+	IsPmtEnabled                                                                                                                                         bool
 	IsType1                                                                                                                                              bool
```

Comment:
Should we remove IsType1 field now? This one is now totally composable of PmtEnabled and Normalcy and we have a public function for it. I believe this field is never used